app/metrics.yaml

```diff
@@ -23,3 +23,100 @@ distribution:
     notification_emails:
       - dmu@mozilla.com
     expires: "2020-05-01"
+url:
+  domains:
+    type: counter
+    lifetime: ping
```

Comment:
Note: `lifetime: ping` is redundant and can be removed here and elsewhere below. It's the default lifetime.